rtrlib/lib/alloc_utils.c

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of RTRlib.
+ *
+ * This file is subject to the terms and conditions of the MIT license.
+ * See the file LICENSE in the top level directory for more details.
+ *
+ * Website: http://rtrlib.realmv6.org/
+ */
+
+#include <stdlib.h>
+
+#include "alloc_utils.h"
+
+void* (*MALLOC_PTR)(size_t size);
```

Comment:
you only mention malloc and free, is there a reason why this would not work for realloc?